addons/website_links/static/src/js/website_links_backend.js

```diff
@@ -0,0 +1,21 @@
+
+odoo.define('web.website_links_embedded', function (require) {
+"use strict";
+var core = require('web.core');
+var Widget = require('web.IFrameWidget');
```

Comment:
Perfect example of bad design: add an iFrame of the frontend inside the backend...